scripts/gallery/numpy/static/css/styles.css

```diff
@@ -0,0 +1,354 @@
+body {
```

Comment:
This file is a copy, that's not desired. Can you instead merge your changes into `static/css/styles.css` or add a new CSS file here with only the rules needed for this Team page and then use both CSS files in the template?

Reply:
The same applies to the NumPy and NumFOCUS logos, we already have them in the repo.

Reply:
And `avatar.png` seems unused so can be removed.

Reply:
> This file is a copy, that's not desired. Can you instead merge your changes into `static/css/styles.css` or add a new CSS file here with only the rules needed for this Team page and then use both CSS files in the template?

ok

Reply:
Will do - this was meant to be a standalone script but I guess after integration we need to remove the duplicates. Working on it.

Reply:
> This file is a copy, that's not desired. Can you instead merge your changes into `static/css/styles.css` or add a new CSS file here with only the rules needed for this Team page and then use both CSS files in the template?

Eliminated.

Reply:
> The same applies to the NumPy and NumFOCUS logos, we already have them in the repo.

Addressed.